handlers/advertisewithus.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 func AdvertiseWithUs(w http.ResponseWriter, r *http.Request) {
-	var advertisewithusTemplate = templates.GetTemplate("layout.html", "advertisewithus.html")
+	var advertisewithusTemplate = templates.GetTemplate(append(layoutTemplateFiles, []string{"advertisewithus.html"}...)...)
```

Comment:
There is no need to create string slice in all the append calls. For the sake of readability, I suggest the following pattern:

```suggestion
	var advertisewithusTemplate = templates.GetTemplate(append(layoutTemplateFiles, "advertisewithus.html", "yetAnother")...)
```

What do you think? This would of course require you to change that for all occurrences of `append(layoutTemplateFiles`, sorry.

Reply:
you are right - fixed for all